frontend/src/components/Output/GoodnessFit.js

```diff
@@ -1,21 +1,42 @@
 import React, {Component} from "react";
 import {observer} from "mobx-react";
 import PropTypes from "prop-types";
+import {ff} from "../../common";
 
 @observer
 class GoodnessFit extends Component {
     render() {
+        const {store} = this.props,
+            goodnessFit = store.goodnessofFit;
```

Comment:
goodnessofFit -> goodnessOfFit; why rename here too `{goodnessofFit} = store` so we know it's the same value

Reply:
changed to gof